sygnal/apnspushkin.py

```diff
@@ -422,25 +422,40 @@ def _get_payload_full(self, n, log):
                 badge = 0
             badge += n.counts.missed_calls
 
-        if badge is not None:
-            aps["badge"] = badge
-
-        if loc_key:
-            aps["mutable-content"] = 1
```

Comment:
is removing this deliberate?

Reply:
Yes sure, clients will decide contents of the `aps` with `default_payload` anymore.